rust-runtime/aws-smithy-http-server/src/rejection.rs

```diff
@@ -262,6 +262,8 @@ convert_to_request_rejection!(std::str::Utf8Error, InvalidUtf8);
 convert_to_request_rejection!(hyper::Error, HttpBody);
 
 // Required in order to accept Lambda HTTP requests using `Router<lambda_http::Body>`.
+#[cfg(feature = "aws-lambda")]
+#[cfg_attr(docsrs, doc(cfg(feature = "aws-lambda")))]
```

Comment:
This provides the feature-flag information next to the type/module in the docs - e.g.
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/20745048/204531545-c7ccae48-e093-46f4-8189-216fc3fdefed.png">